packages/airnode-deployer/src/cli/index.ts

```diff
@@ -158,6 +158,11 @@ yargs(hideBin(process.argv))
       }
     }
   )
+  .example([
+    ['$0 deploy -c myConfig/config.json -s myConfig/secrets.env -r myOutput/receipt.json'],
```

Comment:
```suggestion
    ['$0 deploy -c pathTo/config.json -s pathTo/secrets.env -r myOutput/receipt.json'],
```

reads a bit better

Reply:
:+1: Ill make the corresponding updates to docs as well